spread-shellcheck

```diff
@@ -237,8 +234,19 @@ def main(opts):
 
 
 if __name__ == '__main__':
+    # fiddle with PATH so we pick up a snapped shellcheck first
+    if "/snap/bin" in os.environ["PATH"]:
```

Comment:
It's probably much easier to drop the piece that mangles the PATH and have this instead:
```
SHELLCHECK = os.getenv('SHELLCHECK', 'shellcheck)
...
def checksection(data):
    ...
    subprocess.Popen([SHELLCHECK, "-s", "bash", "-x", "-"]),
                     stdout=subprocess.PIPE,
                     stdin=subprocess.PIPE)
    ...
```

Then `SHELLCHECK=/snap/bin/shellcheck ./spread-shellcheck ...` should call `shellcheck` from snap.